src/engraving/libmscore/stafftype.h

```diff
@@ -392,10 +392,10 @@ class StaffType
 
     // properties setters (setting some props invalidates metrics)
     void  setDurationFontName(const QString&);
-    void  setDurationFontSize(qreal);
+    void setDurationFontSize(qreal);
     void  setDurationFontUserY(qreal val) { _durationFontUserY = val; }
     void  setFretFontName(const QString&);
-    void  setFretFontSize(qreal);
+    void setFretFontSize(qreal);
```

Comment:
why onle those and not any of the others in that block?

Reply:
Should be either all or none. I'M for none

Reply:
fixed